src/stan-users-guide/survival.Rmd

```diff
@@ -0,0 +1,788 @@
+distribution, then its hazard function is constant,
+\begin{eqnarray*}
+h(t \mid \beta)
+& = & \frac{p_T(t \mid \beta)}{S(t \mid \beta)}
+\\[4pt]
+& = & \frac{\textrm{exponential}(t \mid \beta)}{1 - \textrm{exponentialCCDF}(t \mid \beta)}
+\\[4pt]
+& = & \frac{\beta \cdot \exp(-\beta \cdot t)}
+           {1 - (1 - \exp(-\beta \cdot t))}
+\\[4pt]
+& = & \frac{\beta \cdot \exp(-\beta \cdot t)}
+           {\exp(-\beta \cdot t)}
+\\[4pt]
+& = & \beta.
+\end{eqnarray*}
+The exponential distribution is the only distribution of survival
+times with a constant hazard function.
+
+If $T \sim \textrm{Weibull}(\alpha, \sigma),$ then its hazard function
+is
+\begin{eqnarray*}
+h(t \mid \alpha, \sigma)
+& = & \frac{p_T(t \mid \alpha, \sigma)}{S(t \mid \alpha, \sigma)}
+\\[4pt]
+& = & \frac{\textrm{Weibull}(t \mid \alpha, \sigma}{1 - \textrm{WeibullCCDF}(t \mid \alpha, \sigma)}
+\\[4pt]
+& = & 
+\frac{\frac{\alpha}{\sigma} \cdot \left( \frac{t}{\sigma} \right)^{\alpha - 1}
+      \cdot \exp\left(-\left(\frac{t}{\sigma} \right)^\alpha\right)}
+     {1 - \left(1 - 
+       \exp\left(-\left(\frac{t}{\sigma}\right)^\alpha
+     \right)\right)}      
+& = &
+\\[4pt]
+& = & \frac{\alpha}{\sigma}
+  \cdot
+  \left( \frac{t}{\sigma} \right)^{\alpha - 1}.
+\end{eqnarray*}
+
+If $\alpha = 1$ the hazard is constant over time (which also follows
+from the fact that the Weibull distribution reduces to the exponential
+distribution when $\alpha = 1$). When $\alpha > 1,$ the hazard grows as
+time passes, whereas when $\alpha < 1,$ it decreases as time passes.
+
+
+## Proportional hazards model
+
+The exponential model is parametric in that is specifies an explicit
+parametric form for the distribution of survival times.  @cox:1972
+introduced a semi-parametric survival model specified directly in
+terms of a hazard function $h(t)$ rather than in terms of a
+distribution over survival times.  Cox's model is semi-parametric in
+that it does not model the full hazard function, instead modeling only
+the proportional differences in hazards among subjects.
+
+Let $x_n \in \mathbb{R}^K$ be a (row) vector of covariates for subject
+$n$ so that the full covariate data matrix is $x \in \mathbb{R}^{N \times
+K}$.  In Cox's model, the hazard function for subject $n$ is defined
+conditionally in terms of their covariates $x_n$ and the parameter vector
+$\gamma \in \mathbb{R}^K$ as
+\[
+h(t \mid x_n, \beta) = h_0(t) \cdot \exp(x_n \cdot \gamma),
+\]
+where $h_0(t)$ is a shared baseline hazard function and $x_n \cdot
+\gamma = \sum_{k=1}^K x_{n, k} \cdot \beta_k$ is a row vector-vector
+product. 
+
+In the semi-parametric, proportional hazards model, the baseline
+hazard function $h_0(t)$ is not modeled.  This is why it is called
+"semi-parametric."  Only the factor $\exp(x_n \cdot \gamma),$ which
+determines how individual $n$ varies by a proportion from the baseline
+hazard, is modeled.  This is why it's called "proportional hazards."
+
```

Comment:
discuss why not generative w/r/t hazard function but is generative w/r/t ordering.